client/src/pages/EditForm/EditForm.component.tsx

```diff
@@ -69,6 +69,7 @@ const EditForm = ({ match }: EditFormProps): JSX.Element => {
       description: data.postData.description,
       title: data.postData.title,
       tagname: data.tags,
+      topicname: '', //TODO: update when topics selection is introduced in frontend
```

Comment:
If the client type supports using null, you should likewise use null
```suggestion
      topicName: null, //TODO: update when topics selection is introduced in frontend
```